static/sass/styles.scss

```diff
@@ -325,3 +325,8 @@ h2 {
     @extend .p-link--inverted; // sass-lint:disable-line placeholder-in-extend
   }
 }
+
+// Hide images that fail to lazy load due to JS failing or other
+.lazyload {
+  visibility: hidden;
```

Comment:
If using visibility you might want to add `height: 0; width: 0; margin: 0; padding: 0;` otherwise you'll get a bit of empty space.

Reply:
Having the height set explicitly will avoid the page bouncing though.

Reply:
If JS is available... If not the `<noscript>` version will be pushed down. I guess which do you prioritize... Likelihood is most of our sites will not work properly without JS so a bit of empty space isn't the worst issue.